SynergyUnitTests/RandomCodeUnitTests.cs

```diff
@@ -0,0 +1,88 @@
+using MongoDB.Driver;
+using Synergy.Models;
+
+namespace SynergyUnitTests;
+
+public class RandomCodeUnitTests
+{
+    private IMongoDatabase _testDatabase;
+    private IMongoCollection<UserInviteCodes> _randomCodesCollection;
+    private static Random _random = new Random();
+
+    [SetUp]
+    public void Setup()
+    {
+        var client = new MongoClient("mongodb://localhost:27017");
+        _testDatabase = client.GetDatabase("SynergyTest");
+        _randomCodesCollection = _testDatabase.GetCollection <UserInviteCodes>("RandomCodesTest");
+    }
+    
+    [Test]
+    public async Task Test_GenerateRandomCodeInvite()
+    {
+        var teamId = "TEAM_ID";
+        var randomCode = await GenerateRandomCodeInvite(teamId);
+        Assert.That(randomCode.Length, Is.EqualTo(10));
+    }
+
+    public async Task<string> GenerateRandomCodeInvite(string teamId)
```

Comment:
If this is not a tested method this should be a private method so that its easier for other developers to tell whether its meant to be a test or a helper method :)